pressbooks/pressbooks-multi-institution
# fix: map object to `WP_User` class

Fix for pressbooks/private#1476

This PR fixes an issue where when editing an institution, the Institutional Managers assigned to that institution don't show as selected in the page.

**How to test**

1. With the `dev` branch selected, create an institution and assign one or more institutional managers
2. The Institution List should display the assigned institutional managers correctly
3. Click on the edit page. You should not see the Institutional Managers selected in the input
4. Switch to this branch and reload the page. You should now see the Institutional Managers as before
5. Click save and make sure the Institution is updating, persisting the selected Institutional Managers
